### PR TITLE
Bug/GSYE-154: Read/Assign energy, price and rate members of Trade objects

### PR DESCRIPTION
**Pull request by Hassan754**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -46,7 +46,7 @@ deps =
 	-rrequirements/tests.txt
 commands =
     pip install -e .
-	behave --tags=-disabled integration_tests {posargs}
+	behave --tags=-disabled --tags=-slow integration_tests {posargs}
```
Comment (hannesdiedrich) [inline tox.ini:49]: I hope that non of the tests tagged with `slow` failed?

**Review by hannesdiedrich — DISMISSED** [review]
One clarification, LGTM

File: tox.ini (hunk 1 above)
Comment (Hassan754) [reply tox.ini]: @hannesdiedrich nope, I just added this to improve the runtime of the tests, reverted it now

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**